src/main/java/com/univocity/parsers/containers/Csv.java

```diff
@@ -0,0 +1,48 @@
+package com.univocity.parsers.containers;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+/**
+ * @author naveen.kasthuri
+ *
+ * A container that maps column indices to thier names. This avoids having duplicate arrays of
+ * key-value pairs where the keys are the same for every row in the csv file.
+ *
+ * This is used by the Row object and the RowListProcessor. Should not be used otherwise.
+ */
+public class Csv {
```

Comment:
This class name is denotes support for a specific text format, which is not the case. There's no reason why this could not be used by the TSV and Fixed-Width parsers as well. I suggest to rename this class to make it reflect its general purpose (maybe `RecordMetadata` or just `Metadata`)